### PR TITLE
added first homework on js

### PR DESCRIPTION
**Pull request by drsckarasu**

### CODE REVIEW

File: homeworks/homework_24/main.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+// 4. Запросите у пользователя радиус окружности и выведите площадь такой окружности.
+
+let circleRadius = prompt("radius of a circle", "");
+const Pi = 3.14159265359;
+let circleArea = Pi * circleRadius ** 2;
+alert(circleArea);
+
+// 5. Запросите у пользователя расстояние в км между двумя городами и за сколько часов он хочет добраться. Посчитайте скорость, с которой необходимо двигаться, чтобы успеть вовремя.
+ let distance = prompt("distance between cities", "");
```
Comment (idovbush) [inline homeworks/homework_24/main.js:48]: = prompt("distance between cities", ""); if you do not want to pass default value than you can write only prompt("distance between cities")

File: homeworks/homework_24/main.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+let change = total % chockolade;
+alert( `you can buy ${amount} chocolate bars and have change of ${change}`);
+
+// 9. Запросите у пользователя трехзначное число и выведите его задом наперед. Для решения задачи вам понадобится оператор % (остаток от деления).
+
+let number = prompt("enter 3 numbers", "");
+let reverseFirst = number % 10;
+let reverseSecond = ((number % 100) - (number % 10)) / 10;
+let reverseThird = (number - (number % 100)) / 100;
+let result =  String(reverseFirst) + String(reverseSecond) + String(reverseThird);
```
Comment (idovbush) [inline homeworks/homework_24/main.js:80]: did you use this String only for practice?

Comment (drsckarasu) [reply homeworks/homework_24/main.js:80]: yes for first time, it's not correct?

Comment (idovbush) [reply homeworks/homework_24/main.js:80]: as we discussed at the lesson it's not common to do that

**Pull request merged**